.github/workflows/development.yml

```diff
@@ -122,12 +129,17 @@ jobs:
       - name: Build iOS Client
         run: npm run action apple/scripts/ios_package_remote
 
+      - name: Debug iOS build
+        run: ls -R platforms/ios
+      # TODO: upload artifact
+
   android:
     name: Android
     runs-on: ubuntu-20.04
     timeout-minutes: 10
+    environment: Development / Android
     env:
-      ANDROID_SDK_ROOT: /opt/android-sdk
+      ANDROID_SDK_ROOT: ${{ secrets.ANDROID_SDK_ROOT }}
```

Comment:
A couple of weaker reasons:

1) to make use of the production android environment and deployment so that the github behavior is consistent with the ios one
2) could change across different deployments?

Don't feel strongly about it at all.